.env.dev

```diff
@@ -16,6 +16,9 @@ ACCESS_API_BACKEND=http://emulator:8080
 API_URL=http://localhost:3000
 WALLET_DISCOVERY=http://localhost:8701/fcl/authn
 
+WALLET_TOKEN_AMOUNT_FLOW=100.0
+WALLET_TOKEN_AMOUNT_FUSD=100.0
```

Comment:
If I sign up a brand new account in dev environment, my flow and FUSD will be initialized to be 100? if so, I would say we bump this to smth higher (i.e. 10000?). The reason is currently we do not have a very convenience way to add FLOW tokens (have to manually run multiple commands to add tokens to emualtor), so it would be annoying when the developers are running out of the 100 tokens/FUSD and have to figure out how to manually add tokens (if it's their first time doing so)

Reply:
Thanks for the clarification, Alex. The cadence of 100 sounds good to me.